src/main/java/com/careerfair/q/model/db/Student.java

```diff
@@ -2,65 +2,62 @@
 
 import com.careerfair.q.service.database.deserializer.TimestampDeserializer;
 import com.careerfair.q.util.enums.Role;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.google.cloud.Timestamp;
+import com.google.cloud.firestore.annotation.Exclude;
 import com.google.cloud.firestore.annotation.PropertyName;
-import lombok.Getter;
-import lombok.Setter;
 
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
 import java.util.List;
 
-@Getter
-@Setter
 public class Student {
-
     @PropertyName("student_id")
-    @JsonProperty("student_id")
+    @Exclude
```

Comment:
Why did you put @Exclude here?

Reply:
so that the employees and student-id field is not pushed to the database